client/allocrunner/taskrunner/envoy_bootstrap_hook.go

```diff
@@ -373,7 +377,11 @@ func (h *envoyBootstrapHook) Prestart(ctx context.Context, req *ifs.TaskPrestart
 		// Wrap the last error from Consul and set that as our status.
 		_, recoverable := cmdErr.(*exec.ExitError)
 		return structs.NewRecoverableError(
-			fmt.Errorf("error creating bootstrap configuration for Connect proxy sidecar: %v", cmdErr),
+			fmt.Errorf("%w: %v; see: %s",
+				errEnvoyBootstrapError,
+				cmdErr,
+				"<https://www.nomadproject.io/s/envoy-bootstrap-error>",
+			),
```

Comment:
```suggestion
			fmt.Errorf("%w: %v; see: <https://www.nomadproject.io/s/envoy-bootstrap-error>",
				errEnvoyBootstrapError,
				cmdErr,
			),
```

I think we can just put the URL directly in the string template? It doesn't seem to add much to have it as a param.

Reply:
Oh yeah, that was from a previous iteration where I had a variable just for the link.  I'm still a little intrigued by the idea that all doc links in code might be set as variables all in one place, but that's a big tangent I don't want to take right now.